### PR TITLE
Update db command

### PR DESCRIPTION
**Pull request by eyeamnoob**

### CODE REVIEW

**Comment by mhemrg** [comment]
1) Please rename `Hello` to the corresponding class name on each file.
2) We use `I` at the beginning of each interface name not at the end. To be consistent with the rest of the project, please rename your interfaces as well.

**Comment by eyeamnoob** [comment]
Fixed issues that you mentioned

**Comment by mhemrg** [comment]
This is the output of `db -h` command:

```
display commands for working with databases

USAGE
  $ liara db COMMAND

COMMANDS
  db backup  manage backups for a database
  db create  create a new database
  db list    list available databases
  db ls      list available databases
  db off     stop a database
  db on      resize a database
  db remove  remove a database
  db resize  resize a database
  db rm      remove a database
  db start   start a database
  db stop    stop a database

```

Let's remove the `db off` and `db on` commands.

**Comment by mhemrg** [comment]
To create subcommands for the `backup` command, you don't need to do it manually. You should create a directory for it in the `db` directory, like this:

```
db/
    list.js
    create.js
    backup/
        list.js
        create.js
```

**Comment by eyeamnoob** [comment]
Please review again. If there are still issues, let me know.

File: src/commands/db/create.ts [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+      this.error('We do not support germany any more.');
+
+    const hostname = flags.name || (await this.promptHostname());
+    const type = flags.type || (await this.promptType());
+    const version = flags.version || (await this.promptVersion(type));
+    const publicNetwork =
+      flags.network === 'public'
+        ? true
+        : flags.network === 'private'
+        ? false
```
Comment (mhemrg) [inline src/commands/db/create.ts:61]: You can just do this:

```suggestion
    const publicNetwork = flags.network === 'public';
```

File: src/commands/db/backup/list.ts [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+  static aliases = ['db:backup:ls'];
+
+  static flags = {
+    ...Command.flags,
+    name: Flags.string({
+      char: 'n',
+      description: 'name of your database',
+    }),
+  };
```
Comment (mhemrg) [inline src/commands/db/backup/list.ts:33]: Please also add table flags here.

File: src/commands/db/backup/download.ts [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+      }
+      const databaseID = database._id;
+      const backupName = flags.backup || (await this.promptBackupName());
+      const downloadLink = await this.got
+        .post(
+          BackUp.PATH.replace('{database-id}', databaseID) +
+            `/${backupName}/download`
+        )
+        .json<any>();
+      this.log(`download link: ${downloadLink.link}`);
```
Comment (mhemrg) [inline src/commands/db/backup/download.ts:67]: I think this command should download the backup to the current working directory.

File: src/commands/db/backup/list.ts [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        );
+        return {
+          lastModified: `${shamsiData[0]}-${shamsiData[1]}-${shamsiData[2]}`,
+          size: backup.size,
+          name: backup.name,
+        };
+      });
+      const tableConfig = {
+        'last modified': { get: (row: any) => row.lastModified },
+        size: { get: (row: any) => row.size + ' bytes' }, // TODO: output a better format for big size backups
```
Comment (mhemrg) [inline src/commands/db/backup/list.ts:71]: To show this in a better format, you can use the `bytes` package.

File: src/commands/db/create.ts (hunk 1 above)
Comment (eyeamnoob) [reply src/commands/db/create.ts:61]: In that way, anything other than 'public' is considered as private. Is it OK?

Comment (mhemrg) [reply src/commands/db/create.ts:61]: oclif flags can have a limited allowed list of values. checkout the `builder-location` flag in the `deploy` command. it only accepts `iran | germany` as values.

**Comment by mhemrg** [comment]
After merging this, we will release a beta version so we can use these new commands before launching them.

**Comment by eyeamnoob** [comment]
At least i can fix my bugs😅

Mohammad Hossein Ebadollahi ***@***.***> schrieb am Mi., 27.
Sept. 2023, 16:07:

**Comment by mhemrg** [comment]
Please make sure to do the requested changes.

**Comment by eyeamnoob** [comment]
I downloaded the backup file, included the table flags in the db:backup:list command, and utilized the bytes package. What other tasks do I need to complete? I simply did not implement the suggestion for the db:create command regarding network flags. This is because I would like to prompt the user for the network if they do not provide the --network flag.

**Pull request merged**